src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        Manufacturer manufacturer = new Manufacturer("BMW", "USA");
+        manufacturerService.create(manufacturer);
+        Driver firstDriver = new Driver("Alex", "123");
+        Driver seconfDriver = new Driver("Pavel", "456");
+        driverService.create(firstDriver);
+        driverService.create(seconfDriver);
+        System.out.println(driverService.getAll());
+
+        Car car = new Car();
+        car.setModel("Mercedes");
+        car.setManufacturer(manufacturer);
+        car.setDrivers(List.of(firstDriver, seconfDriver));
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        car.setModel("Skoda");
+        car = carService.get(1L);
```

Comment:
Let's try to not hardcode id values in the code, because it can be changed.
```suggestion
        car = carService.get(car.getId());
```
Please fix other cases in the code.